in_text/text_data_release.yml

```
@@ -309,63 +315,63 @@ entities:
   -
     data-name: 03_model/out/Run_Manuscript_Results/inputs.zarr
     data-description: >-
-      XX
+      time series of inputs for the model
   -
     data-name: 03_model/out/Run_Manuscript_Results/target.zarr
     data-description: >-
-      XX
+      time series of the modeling target (salt front location)
   -
     data-name: 03_model/out/Run_Manuscript_Results/prepped_model_io_data
     data-description: >-
-      XX
+      inputs and targets merged and prepared for model training
   -
     data-name: 03_model/out/Run_Manuscript_Results/**/ModelResults.csv
     data-description: >-
-      XX
+      daily prediction of 7-day average salt front location for each model replicate
     attributes: 
     -
       attr-label: saltfront_obs
       attr-def: >-
-        XX
+        observed daily 7-day average location of the salt front 
```

where do observations come from? DRBC?
```suggestion
        Observed daily 7-day backward-looking averagesalt front location (source: Delaware River Basin Commission, 2023).
```